src/main/java/org/broadinstitute/hellbender/engine/TwoPassReadWalker.java

```diff
@@ -0,0 +1,96 @@
+package org.broadinstitute.hellbender.engine;
```

Comment:
The reason it is in the engine is because many of the features that the twoPassWalker needs to access in order to operate are package protected within the engine code.